Samples/Samples.Cards/Program.cs

```diff
@@ -22,7 +25,23 @@ public static void Main(string[] args)
         // Bind the application to localhost:3978 when run with `dotnet run`
         builder.WebHost.UseUrls("http://localhost:3978");
         builder.Services.AddOpenApi();
-        builder.Services.AddTransient<Controller>();
+
+        // Use environment variable to choose between Controller and Minimal API approaches
+        var useMinimalApi = builder.Configuration.GetValue<bool>("USE_MINIMAL_API", false);
```

Comment:
I think the samples should focus explaining one concept, I would add another sample instead of adding a config to select one or the other.